apps/explorer/src/pages/validator/ValidatorDetails.tsx

```diff
@@ -77,6 +86,21 @@ function ValidatorDetails() {
                     tallyingScore={tallyingScore}
                 />
             </div>
+            {atRiskRemainingEpochs !== null && (
+                <div className="mt-5">
+                    <Banner fullWidth variant="error">
+                        <div>
```

Comment:
do we need to use the shared `Text` components for rendering this text or no?

Reply:
It does not matter really since the text styling is being overridden by the Banner anyway for the content.